### PR TITLE
python2- prefix for fedora 26 packages

### PR DESCRIPTION
**Pull request by gtmanfred**
### What does this PR do?
The package name was changed in Fedora 26

### Tests written?

Yes

### CODE REVIEW

**Review by Ch3LL — APPROVED** [review]

**Comment by cachedout** [comment]
Go Go Jenkins!

**Review by terminalmage — APPROVED** [review]

**Comment by cachedout** [comment]
Go Go Jenkins!

File: salt/client/__init__.py [hunk 1]
```diff
@@ -738,7 +738,7 @@ def cmd(self,
                         ret[mid] = (data if full_return
                                 else data.get('ret', {}))
 
-            for failed in list(set(pub_data['minions']) ^ set(ret)):
+            for failed in list(set(pub_data['minions']) - set(ret)):
```
Comment (cachedout) [inline salt/client/__init__.py]: Eh? What is this doing in a PR about package naming? :]

**Review by cachedout — CHANGES_REQUESTED** [review]

File: salt/client/__init__.py (hunk 1 above)
Comment (gtmanfred) [inline salt/client/__init__.py]: yup, not sure how that ended up on the top of this.

fixed.

**Review by rallytime — APPROVED** [review]

**Pull request merged**